packages/dapp/src/hooks/useKeysManager.ts

```diff
@@ -0,0 +1,212 @@
+import type { VerificationMethodReference } from '@windingtree/org.json-schema/types/org.json';
+import type { AnySchema } from '@windingtree/org.id-utils/dist/object';
+import type { GenericStateRecord } from '../store';
+import { useState, useCallback, useEffect } from 'react';
+import { useAppReducer } from '../store/reducer';
+
+import { object, uid } from '@windingtree/org.id-utils';
+import { org } from '@windingtree/org.json-schema';
+
+export const revocationReasonValues = object.getDeepValue(org, 'definitions.VerificationMethodRevocationReference.properties.reason.enum') as string[];
+export type RevocationReference = typeof revocationReasonValues[number]
+
+export const allowedKeysTypes: string[] = ['EcdsaSecp256k1VerificationKey2019'];
+
+export type KeyType =
+	typeof allowedKeysTypes[number];
+
+const keyRecordSchema: AnySchema = {
```

Comment:
It is better always to export all definitions because these definitions may be reused somewhere.

Reply:
✔️